site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -124,7 +141,45 @@
         </div>
       </div>
     {{ form_end(form) }}
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Delete</h3>
+    </div>
+    <div class="panel-button" id="delete-panel-button">
+      <br>
+      <a href="#" id="delete-panel-button" class="btn btn-default" data-toggle="modal" data-target="#delete-modal"><strong>Delete this project</strong></a>
```

Comment:
Couldn't this be a bootstrap `btn-danger`?

Reply:
I set the id to btn-danger, see if it looks better. As i changed the id, i think the margin is  not required. I will remove that as well.